docs/src/adr/NNNadr-template.md

```diff
@@ -0,0 +1,48 @@
+---
+authors: Octavia Buttler, Eduardo Galleano
+last revised: YYYY-MM-DD
+---
+
+# ADR-{NNN}: {Decision Subject}
```

Comment:
Why is this template part included in the PR?

Reply:
I just based this off of the template PR, so I need to rebase.